### ISSUE TITLE
Enable iNav Radar (Lora) displays on ALL OSD pages 

### ISSUE THREAD

**Issue by Pairan** [iNavFlight/inav#5073]
## Current Behavior
Perhaps I missed something but it seems the LoRa Radar only display the other POIs in the Default screen. Further it is hard to figure, what is needed to enable / disable it.

## Desired Behavior
It would be nice if we could have a flag defining the "Radar POIs" as enabled/disabled PER OSD so the data could be shown better instead of colliding with the other OSD elements. 

## Suggested Solution
- add an OSD item id (or even more as the visibility of the home point is changeable too)
- enable this new "item id" in all configureable OSD screens
- make these changes accessable in the configurator too

## Who does this impact? Who is this for?
With this enhancement it would be easy to configure the OSD more tidy. You may want to see the default screen with all mainly desired items but I turns messy pretty fast as **POI A** and or **POI B** enter the screen and collides with other items like altitude or speed. 

By configuring an additional OSD layout without so many items it would be shown more clear and accompany the map / radar item almost perfectly.

**TLDR;**
In case your main OSD screen is to populated for that stuff, you could simply switch over to a different layout having the iNav Radar feature displayed in a cleaner way with less collisions.

I really love the possibility to see my buddies in the OSD - so far it works good but I would like to make it a bit better by having it on several OSD layouts.
